### PR TITLE
docs:update install docs

### PR DESCRIPTION
**Pull request by ZeroMagic**
update install docs

### CODE REVIEW

File: docs/deploy.md [hunk 1]
```diff
@@ -103,13 +103,24 @@ EOF
...
 ```sh
-docker run --rm -d -p 3000:3000 grafana/grafana
+mkdir -p /tmp/provisioning/datasources
+cat >/tmp/provisioning/datasources/prometheus.yaml <<EOF
+apiVersion: 1
+datasources:
+  - name: prometheus
+    type: prometheus
+    access: direct
+    url: http://localhost:9090
```
Comment (mozhuli) [inline docs/deploy.md]: use `<Your-HostIP>` replace `localhost` should better

File: README.md [hunk 2]
```diff
@@ -54,9 +54,20 @@ scrape_configs:
     static_configs:
       - targets: ['127.0.0.1:9091']
```
Comment (mozhuli) [inline README.md]: use `<Your-HostIP>` replace `127.0.0.1` should better

File: README.md [hunk 3]
```diff
@@ -54,9 +54,20 @@ scrape_configs:
...
 # install Grafana
-docker run --rm -d -p 3000:3000 grafana/grafana
+mkdir -p /tmp/provisioning/datasources
+cat >/tmp/provisioning/datasources/prometheus.yaml <<EOF
+apiVersion: 1
+datasources:
+  - name: prometheus
+    type: prometheus
+    access: direct
+    url: http://localhost:9090
```
Comment (mozhuli) [inline README.md]: use `<Your-HostIP>` replace `localhost` should better

**Comment by ZeroMagic** [comment]
Now I have replaced.

**Comment by mozhuli** [comment]
LGTM

**Pull request merged**